apps/tx-builder/src/components/Dashboard.tsx

```diff
@@ -157,6 +165,19 @@ const Dashboard = (): ReactElement => {
           <CreateNewBatchCard />
         )}
       </TransactionsSectionWrapper>
+
+      {editingTransactionIndex != null && (
+        <EditTransactionModal
```

Comment:
What is your opinion about to move this `EditTransactionModal`  Component to the `TransactionsBatchList`?

I think that its a good idea because we will need it also in the Review and Confirm page:

![Captura de pantalla 2022-02-21 a las 14 50 37](https://user-images.githubusercontent.com/26763673/154967954-4ed02338-a570-4e25-87c4-cdf1ebb08647.png)